apps/forge/src/routes/test.tsx

```diff
@@ -0,0 +1,13 @@
+import { createFileRoute } from '@tanstack/react-router'
+import { Input } from '@ui/components/ui/input'
+import { Label } from '@ui/components/ui/label'
+
+export const Route = createFileRoute('/test')({
+  component: RouteComponent,
+})
+
+function RouteComponent() {
+  return <Label>
+    Text
+  </Label>
```

Comment:
Looks great, preferred the Input component though
```suggestion
  return <Input></Input>
```